triton_rocm_20-52.Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM rocm/pytorch:rocm5.2.3_ubuntu20.04_py3.7_pytorch_1.12.1
```

Comment:
maybe upgrade this to something like:

rocm/pytorch:rocm5.3.2_ubuntu20.04_py3.8_1.12.1

Reply:
if this can be changed, suggest to keep `rocm/pytorch:latest`, which will be updated automatically.